src/encoded/static/vis_defs/PCS_vis_def.json

```diff
@@ -0,0 +1,51 @@
+{
+    "PCS": {
+        "rule": "'{assay_term_name}' == 'pooled clone sequencing'",
+        "assay_composite": {
+            "longLabel":  "Collection of ENCODE pooled clone sequencing experiments",
+            "shortLabel": "ENCODE pooled clone sequencing"
+        },
+        "longLabel":  "{assay_title} of {replicates.library.biosample.summary} - {accession}",
+        "shortLabel": "{assay_title} of {biosample_term_name} {accession}",
+        "sortOrder": [ "Biosample", "Replicates", "Views" ],
+        "Views":  {
+            "tag": "view",
+            "group_order": [ "Signal of unique reads" ],
+            "groups": {
+                "Signal of unique reads": {
+                    "tag": "aSIG",
+                    "visibility": "full",
+                    "type": "bigWig",
+                    "viewLimits": "0:10",
+                    "autoScale": "off",
+                    "maxHeightPixels": "64:18:8",
+                    "windowingFunction": "mean+whiskers",
+                    "output_type": [ "signal of unique reads" ]
+                }
+            }
+        },
+        "other_groups":  {
+            "dimensions": { "Biosample": "dimY", "Replicates": "dimA" },
+            "dimensionAchecked": "first",
+            "groups": {
+                "Replicates": {
+                    "tag": "REP",
+                    "groups": {
+                        "replicate": {
+                            "title_mask": "Replicate_{replicate_number}",
+                            "combined_title": "Pooled"
+                        }
+                    }
+                },
+                "Biosample": {
+                    "tag": "BS",
+                    "groups": { "one": { "title_mask": "{biosample_term_name}"} }
+                }
```

Comment:
Isn't that pooled clone sequencing typically don't have real biosample?

Reply:
You are right they do not have a real biosample in the conventional sense. They aren't cell free though. Are you suggesting we remove this group? (also biosample info in the long and short labels?)